packages/ui/vite.config.ts

```diff
@@ -20,5 +20,8 @@ export default defineConfig({
   ],
   resolve: {
     preserveSymlinks: true // this is the fix!
+  },
+  optimizeDeps: {
+    include: ['@mui/icons-material']
   }
```

Comment:
This is applying a trick that worked for me from https://github.com/mui/material-ui/issues/32727#issuecomment-1553100858